test/.gitignore

```diff
@@ -1,2 +1,5 @@
 credentials.conf
-tmp_*
\ No newline at end of file
+tmp_*
```

Comment:
No more newline?

Reply:
OK. It wasn't intentional!

Reply:
The original file (on the left here) had no newline because `tmp_*` was the last line in the file. 
The new file adds a newline because it also adds more lines to the file.